src/types/transaction.ts

```diff
@@ -136,17 +139,20 @@ export type ContextSummaryVariableType =
       value?: string;
       indexed?: boolean;
       emphasis?: boolean;
+      units?: string;
```

Comment:
Let's make unit required for this
```suggestion
      unit: string;
```